packages/hash/api/src/graphql/resolvers/comments/commentProperties.ts

```diff
@@ -0,0 +1,29 @@
+import { ApolloError } from "apollo-server-express";
+import { Comment, UnresolvedGQLComment } from "../../../model";
+import {
+  CommentProperties as GQLCommentProperties,
+  ResolverFn,
+} from "../../apiTypes.gen";
+import { GraphQLContext } from "../../context";
+
+export const commentProperties: ResolverFn<
```

Comment:
**Blocking:** I could be missing something, but I don't think that this resolver should exist. The three properties that are defined on the `CommentProperties` GQL type are all stored in the `properties` object in the datastore. Therefore I don't think this field resolver doesn't serve a purpose.

Reply:
Removed the resolver in 9affaa36c8a93447844957915a8d38297236105c.